### DISCUSSION TITLE
Allow rearranging of settings sections

### DISCUSSION THREAD

**Discussion by eggooooo** [ppy/osu#21274] [Ideas] [answered]
Having used stable for a while now, i can't seem to get used to lazer's settings layout (muscle memory etc.)
In my opinion, the option to rearrange settings to the users preference would be a pretty neat thing to have

My suggestion would be having the icons in the options sidebar be "drag and drop"-able

**Comment by peppy** [comment, marked as answer]
I think this is going too far in terms of customisation, but I do think we will add "pinned settings", which will show in one group at the top. Would that be enough to help you out?
